tail/tail.go

```diff
@@ -135,7 +218,7 @@ func Tail(ctx context.Context, logger log.Logger, dir string, promMon *prometheu
 	} else {
 		level.Info(logger).Log(
 			"msg", "starting from checkpoint",
-			"directory", dir,
+			"directory", cpdir,
```

Comment:
Thank you (for fixing my mistake)! Would you please apply one more of these dir->cpdir fixes, on line 212? 🥇

Reply:
I didn't update this for 212 as ErrNotFound is set when there's no checkpoint directory